### DISCUSSION TITLE
Register through oidc sends verfification link to e-mail

### DISCUSSION THREAD

**Discussion by shreeharsha-factly** [ory/kratos#2378] [Q&A] [answered]
Once I register with OIDC ideally email should be verified but getting a verification link. Is it expected behavior or not?

**Comment by aeneasr** [comment, marked as answer]
There’s currently no way to tell Kratos that an email is already verified in the OIDC JsonNet. I’m not sure if there’s an issue for it already, but if there isn’t please feel free to add one :)

**Reply by Benehiko** [reply]
There is an issue about this already :)
https://github.com/ory/kratos/issues/1057

**Reply by shreeharsha-factly** [reply]
@Benehiko thanks

**Comment by Benehiko** [comment]
I think this is a duplicate discussion of https://github.com/ory/kratos/discussions/1045
